celery/utils/time.py

```diff
@@ -218,12 +218,15 @@ def remaining(
         ~datetime.timedelta: Remaining time.
     """
     now = now or datetime.utcnow()
```

Comment:
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow is deprecated in Python 3.12